.github/workflows/ci.yml

```diff
@@ -72,3 +72,21 @@ jobs:
         run: |
           PATH=$PATH:$PWD/tools/spell
           for lexfile in `find . -name lexicon.txt`; do dir=${lexfile%/lexicon.txt}; find-unknown-comment-words --directory $dir; if [ $? -ne "0" ]; then exit 1; fi; done;
+  fmt:
```

Comment:
Wouldn't "formatting-check" be more descriptive?

Reply:
Changed to formatting. Check will be redundant with the workflow name.

Reply:
Was recommending in reference to the `spell-check` job name, though I agree with your point of redundancy